pypeit/core/wavecal/autoid.py

```diff
@@ -1772,11 +1774,10 @@ def run_brute(self, min_nlines=10):
 
         # Now that all slits have been inspected, cross match to generate a
         # list of all lines in every slit, and refit all spectra
-        if self._nslit > 1:
+        obad_slits = self.cross_match(good_fit, self._det_weak) if self._ok_mask.size > 1 else np.array([])
+        if obad_slits.size > 1:
             msgs.info('Checking wavelength solution by cross-correlating with all slits')
-
             msgs.info('Cross-correlation iteration #1')
```

Comment:
There are several options here, but I think the two information messages should be put above `obad_slits = self.cross_match(...) blah`. Alternatively, the second info message could be removed and change `cntr=1`. Otherwise, there appears to be printout that says `iteration 1` and then immediately `iteration 2`

Reply:
Looks good to me!